tools/lint-js.js

```diff
@@ -139,8 +139,7 @@ if (cluster.isMaster) {
       printProgress();
       outFn('\r\n');
     }
-    if (code === 0)
-      process.exit(failures ? 1 : 0);
+    if (code === 0) process.exit(failures ? 1 : 0);
```

Comment:
One here yet.

Reply:
About to board my plane so other edits will have to wait until I get home (unless my plane has internet).